step-release-vis/src/app/components/environment/environment.html

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+  <app-polygon
+    *ngFor="let polygon of polygons"
+    [polygon]="polygon"
+  >
+  </app-polygon>
```

Comment:
Not sure if this approach will work. As i described in [this](https://github.com/googleinterns/step251-2020/pull/100#discussion_r470028677) comment, having a poly as a separate component wraps it's template contents in a selector tag, which breaks the SVG.